label_studio/tests/data_manager/test_api_actions.py

```diff
@@ -16,7 +16,7 @@
     ],
 )
 @pytest.mark.django_db
-def test_action_delete_all_tasks(tasks_count, annotations_count, predictions_count, business_client, project_id):
+def test_action_delete_all_tasks(tasks_count, annotations_count, predictions_count, business_client, project_id):  # noqa: F811
```

Comment:
I think it'd be good to disable F811 for pytest files (this can be a followup though)

Reply:
yeah also that project_id is a fixture which ruff has no way of detecting.  I'm fine with just disabling it